harness/determined/cli/_util.py

```diff
@@ -99,6 +100,22 @@ def setup_session(args: argparse.Namespace) -> api.Session:
     )
 
 
+def session(
```

Comment:
If this decorator is a necessary part of having persistent HTTP sessions, I haven't figured out the mechanism.